cpp/zotaut/journal_timeliness_checker.cc

```diff
@@ -91,9 +94,10 @@ int Main(int argc, char *argv[]) {
         if (section.find("user_agent") != section.end())
             continue; // Not a journal section.
 
-        const auto delivery_mode(section.getEnum("zotero_delivery_mode", ZoteroHarvester::Config::STRING_TO_UPLOAD_OPERATION_MAP,
-                                                 ZoteroHarvester::Config::UploadOperation::NONE));
-        if (delivery_mode != ZoteroHarvester::Config::UploadOperation::LIVE or section.getBool("zeder_newly_synced_entry", false))
+        const auto delivery_mode(static_cast<ZoteroHarvester::Config::UploadOperation>(
+             section.getEnum("zotero_delivery_mode", ZoteroHarvester::Config::STRING_TO_UPLOAD_OPERATION_MAP,
+                             ZoteroHarvester::Config::UploadOperation::NONE)));
+        if (section.getBool("zeder_newly_synced_entry", false))
```

Comment:
Is this really correct? Shouldn't we skip entries that are newly synced, instead of processing only them?
(and if the delivery_mode has no effect on this condition, we might as well perform this check before getting the delivery mode)

Reply:
Newly synced are skipped - the `false` is the default value to return. I left the determination of the delivery at its previous place to reduce merge conflicts with the existing selective evaluation code if it has to be reintegrated